### PR TITLE
docs: add readme

### PR DESCRIPTION
**Pull request by ischasny**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,20 @@
-# era-observability
-Grafana dashboards for zkSync Era
+# Era Observability
+
+This repository contains json sources for Grafana dashboards that are used to observe `ZKSync Era` as well as can be used for any other hyperchain deployment. Dashboards are regularly updated.
```
Comment (shahar4) [inline README.md]: nit: `ZKSync Era` -> `zkSync Era`

File: README.md [hunk 2]
```diff
@@ -1,2 +1,20 @@
...
-Grafana dashboards for zkSync Era
+# Era Observability
+
+This repository contains json sources for Grafana dashboards that are used to observe `ZKSync Era` as well as can be used for any other hyperchain deployment. Dashboards are regularly updated.   
+
+## How to use
+
+This section goes through a simple setup for a local deployment.
+
+1. Setup your local environment and start services as described [here](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/launch.md). `zk init` and `zk server` is enough for the matter of this exercise;
```
Comment (shahar4) [inline README.md]: nit: `Setup` -> `Set up` here and below

File: README.md [hunk 3]
```diff
@@ -1,2 +1,20 @@
...
+3. Open the Grafana UI (`http://127.0.0.1:3000/` if you used `Dockprom`) and import the dashboards via regular json import;
+4. (Optionally) run the integration tests via `zk test i server` to generate some transactions.
+
+Voila! You should see some data displayed in Grafana now.
+
+If some charts do not work please raise an issue in this repo. 
+
+## Dashboards
+
+* **General** - one pager high level overview of L1/L2 blockchains and L2 APIs;
```
Comment (shahar4) [inline README.md:18]: nit: add Sequencer.json ?

File: README.md [hunk 4]
```diff
@@ -1,2 +1,20 @@
-# era-observability
-Grafana dashboards for zkSync Era
+# Era Observability
+
+This repository contains json sources for Grafana dashboards that are used to observe `ZKSync Era` as well as can be used for any other hyperchain deployment. Dashboards are regularly updated.   
+
+## How to use
```
Comment (shahar4) [inline README.md:6]: Should you include `--run-observability` flag here?

File: README.md [hunk 5]
```diff
@@ -1,2 +1,20 @@
...
+1. Setup your local environment and start services as described [here](https://github.com/matter-labs/zksync-era/blob/main/docs/guides/launch.md). `zk init` and `zk server` is enough for the matter of this exercise; 
+2. Setup local Prometheus + Grafana, for example using [Dockprom](https://github.com/stefanprodan/dockprom);
+3. Open the Grafana UI (`http://127.0.0.1:3000/` if you used `Dockprom`) and import the dashboards via regular json import;
+4. (Optionally) run the integration tests via `zk test i server` to generate some transactions.
+
+Voila! You should see some data displayed in Grafana now.
+
+If some charts do not work please raise an issue in this repo. 
+
+## Dashboards
```
Comment (shahar4) [inline README.md:16]: Might be worth flagging
- some panels might be empty depending on configuration (as mentioned in the Slack message)
- this repo is still WIP and more dashboards are to come

File: README.md (hunk 4 above)
Comment (ischasny) [reply README.md:6]: Good point. Rewrote the section given that now we have `--run-observabilit` in place.

**Review by shahar4 — DISMISSED** [review]

**Review by shahar4 — APPROVED** [review]

**Pull request merged**